.github/workflows/windows-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e tests on Windows
+
+on:
+  schedule:
+    - cron: '13 5 * * 1-5'
+  workflow_dispatch: {}
+defaults:
+  run:
+    shell: powershell
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 90
+    runs-on: [self-hosted, Windows, X64]
+    steps:
+      - run: wsl --unregister rancher-desktop
+        continue-on-error: true
+      - run: wsl --unregister rancher-desktop-data
+        continue-on-error: true
+      - run: >
+          Remove-Item -Path "~/AppData/Local/rancher-desktop" -Recurse -Force -ErrorAction Ignore
```

Comment:
why did you replace $HOME with ~ ?

Reply:
It's shorter, and more common in Unix usage; I can switch it back.

Reply:
It is a shell feature, and on Unix it doesn't actually expand when inside a quoted string:

```console
$ echo ~/suse
/Users/jan/suse

$ echo "~/suse"
~/suse
```

Not sure what Powershell does, but seeing it inside quotes makes me pause...